restapi/src/main/java/io/stargate/web/docsapi/service/ReactiveDocumentService.java

```diff
@@ -660,6 +660,7 @@ private void writeNewArrayState(
         bindParams,
         processedPath,
         timeSource.currentTimeMicros(),
+        null,
```

Comment:
is this intended? What actually about the documents we insert with TTL, but then don't update with TTL.. This smells like a problem imo?

Reply:
yea I think I need to allow supplying a TTL to this as well